src/project/qml/MuseScore/Project/internal/MeasuresSettings.qml

```diff
@@ -46,15 +46,15 @@ FlatButton {
 
         property string pickupMessage: {
             if (withPickupMeasure.checked) {
-                return qsTrc("project", "Pickup: ") +
+                return qsTrc("project", "Pickup:") + " " +
                         model.pickupTimeSignature.numerator + "/" + model.pickupTimeSignature.denominator
             }
 
             return qsTrc("project", "No pickup")
         }
 
         font: ui.theme.largeBodyFont
-        text: model.measureCount + qsTrc("project", " measures, \n") + pickupMessage
+        text: model.measureCount + qsTrc("project", " measures.") + "\n" + pickupMessage
```

Comment:
This should be
```suggestion
        text: qsTrc("project", "%1 measures").arg(model.measureCount) + ", \n" + pickupMessage
```

Reply:
OK. I'll leave the comma in the translatable string, for LTR languages

Reply:
done